src/main/java/com/iexec/common/sgx/SgxDriverMode.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2022 IEXEC BLOCKCHAIN TECH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.iexec.common.sgx;
+
+public enum SgxDriverMode {
+    NONE(new String[]{}),
+    LEGACY(new String[]{"isgx"}),
+    NATIVE(new String[]{"sgx_enclave", "sgx_provision"});
+
+    private final String[] devices;
+
+    SgxDriverMode(String[] driverNames) {
+        this.devices = driverNames;
+    }
+
+    public String[] getDevices() {
+        return devices;
+    }
+
+    /**
+     * Returns {@literal false} if given {@link SgxDriverMode} is {@literal null}
+     * or {@link SgxDriverMode#NONE}, {@literal true} otherwise.
+     *
+     * @param driverMode {@link SgxDriverMode} object to check.
+     * @return {@literal false} if given {@link SgxDriverMode} is {@literal null}
+     * or {@link SgxDriverMode#NONE}, {@literal true} otherwise.
+     */
+    public static boolean isDriverModeNotNone(SgxDriverMode driverMode) {
+        return driverMode != null && !driverMode.equals(NONE);
```

Comment:
```suggestion
        return driverMode != null && driverMode != NONE;
```
It seems better to use `!=` operator instead of `equals`
https://www.baeldung.com/a-guide-to-java-enums

Reply:
Done in https://github.com/iExecBlockchainComputing/iexec-common/pull/308/commits/24b0c04479f5ce6a5963a0c09882a81f478e7c41.